fortio_main.go

```diff
@@ -525,7 +526,7 @@ func grpcClient() {
 	}
 	httpOpts := bincommon.SharedHTTPOptions()
 	if *doHealthFlag {
-		status, err := fgrpc.GrpcHealthCheck(host, *healthSvcFlag, count, &httpOpts.TLSOptions)
+		status, err := fgrpc.GrpcHealthCheck(host, *healthSvcFlag, count, &httpOpts.TLSOptions, grpcMD())
```

Comment:
pass httpOpts.ExtraHeaders instead? (SharedHTTPOptions() is already called once just above)